### DISCUSSION TITLE
Where to file Python string highlighting bugs?

### DISCUSSION THREAD

**Discussion by nschloe** [NvChad/NvChad#2495] [Q&A] [answered]
I've noticed a couple of bugs when it comes to Python string highlighting:

- `r` (raw) prefixes work well, but not when combined with f-string as as `fr` prefix.
- Unicode literals are not highlighted if the description contains a dash.

![screenshot_2023-11-15-091750](https://github.com/NvChad/NvChad/assets/181628/b43c1f3f-b5ef-4944-abb9-700b002a7635)

I'm not sure where to file those bugs. Any hints?

**Comment by siduck** [comment, marked as answer]
run Inspect command to get the highlight groups, make sure python tsparser is installed


https://github.com/NvChad/NvChad/assets/59060246/93b0ebf4-f3fc-4abd-83a8-9cddc67e1564

**Reply by nschloe** [reply]
Great, thanks!
